main.go

```diff
@@ -156,3 +156,26 @@ func registerCapabilities(port string, log logger.Handler) {
 		log.Info(err.Error())
 	}
 }
+
+func registerDynamicCapabilities(port string, log logger.Handler) {
+	registerWorkloads(port, log)
+	//Start the ticker
+	const reRegisterAfter = 24
+	ticker := time.NewTicker(reRegisterAfter * time.Hour)
+	for {
+		select {
+		case <-ticker.C:
+			registerWorkloads(port, log)
+		}
+	}
+
+}
+func registerWorkloads(port string, log logger.Handler) {
+	log.Info("Registering latest workload components")
+	// Register workloads
+	if err := oam.RegisterWorkLoadsDynamically(mesheryServerAddress(), serviceAddress()+":"+port); err != nil {
+		log.Info(err.Error())
```

Comment:
It’s important for a user to understand that their adapter is malfunctioning. What does it take to sent this as an event to the UI?

Reply:
> The Tests check was failing previously, but the other failed checks is new. Let’s get them all green, if you can.

Yups, in next commit, it would be fixed.

Reply:
> It’s important for a user to understand that their adapter is malfunctioning. What does it take to sent this as an event to the UI?

Should this be considered a malfunction from user standpoint?  I mean, in the case where we will have seeded content and no adapter registering for it-yes it should be reported to the user. Here, as far as user is concerned, the capabilities haven't been registered.
With the seed content PR ,we will also make some additional checks in meshery server and send the user some error like-"Capability is not registered for this pattern" or something like that. There we can attach one of the probable cause as "adapter might have failed to register few capabilities (could not found proper version) "/or something like that. Checking the adapter logs will confirm the issue.

Reply:
This consideration is orthogonal to seed content. Going forward PatternOps (pattern-based operations) will be used for all of the adapter operations. If an adapter fails to register is capabilities (register its operations), we should denote this in the UI. For now, a warning level event can be sent, so the user can see this in the Notification Center.